static/styles/dark_theme.css

```diff
@@ -581,6 +581,25 @@ body.dark-theme {
         background-color: hsla(0, 0%, 0%, 0.2);
     }
 
+    .stream-row {
```

Comment:
Can you add a comment for this section explaining that this is how we implement coloring these SVGs.

Reply:
I've added comments mentioning what each section is responsible for.